include/rocksdb/wide_columns.h

```diff
@@ -221,8 +221,26 @@ inline bool operator!=(const PinnableWideColumns& lhs,
 }
 
 // Class representing attribute group. Attribute group is a logical grouping of
-// wide-column entities by leveraging Column Families. Wide-columns returned
-// from the query are pinnable.
+// wide-column entities by leveraging Column Families.
+// Used in Write Path
+class AttributeGroup {
+ public:
+  ColumnFamilyHandle* column_family() const { return column_family_; }
+  const WideColumns& columns() const { return columns_; }
```

Comment:
Since this class is used on the write path, it might make sense to add non-const accessors (e.g. `WideColumns& columns()`) as well so that clients can modify the structure before the write if needed. This would enable them to do things like building the attribute group gradually, for example by using `ag.columns().emplace_back(...);`.